tools/aequatus/aequatus-generator.py

```diff
@@ -35,15 +54,61 @@ def gene_to_json(fname):
 
 
 def join_json(tree_dict, gene_dict, cigar_dict):
+    global version
+
     aequatus_dict = {'tree': tree_dict,
                      'member': gene_dict}
 
     if cigar_dict:
         aequatus_dict['cigar'] = cigar_dict
+
+    aequatus_dict['ref'] = refGene
+    aequatus_dict['protein_id'] = refProtein
+    aequatus_dict['transcript_id'] = refTrancript
+    aequatus_dict['version'] = version
+
     return aequatus_dict
 
 
+def parseGeneJSON(obj, id):
+    global refGene
+    global refTrancript
+
+    if "Transcript" in obj:
+        for each in obj["Transcript"]:
+            if "Translation" in each:
+                if each["Translation"]["id"] == id:
+                    refGene = obj["id"]
+                    refTrancript = each["id"]
+
+
+def getProteinIDfromGeneJSON(id):
+    global gene_dict
+    for obj in gene_dict:
+        if "Transcript" in gene_dict[obj]:
+            for each in gene_dict[obj]["Transcript"]:
+                if each["id"] == id:
+                    return each["Translation"]["id"]
+
+
+def parseTree(obj):
+    global refProtein
+    global refGene
+    global refTrancript
+    if "children" not in obj:
+        jsonArrayGene.append(obj["id"]["accession"])
+        jsonArrayTranscript.append(obj["sequence"]["id"][0]["accession"])
```

Comment:
The `jsonArrayGene` and `jsonArrayTranscript` lists are not used after being created, can be deleted?